code/modules/jobs/job_types/captain.dm

```diff
@@ -68,21 +68,21 @@ Head of Personnel
 	minimal_player_age = 10
 	exp_requirements = 180
 	exp_type = EXP_TYPE_CREW
-	exp_type_department = EXP_TYPE_SUPPLY
+	exp_type_department = EXP_TYPE_SERVICE

```

Comment:
tfw only clown mains can play HoP now

Reply:
you got it bby

Reply:
although I do like the idea of only clown mains being allowed to be HoP